src/java.base/share/classes/java/util/Properties.java

```diff
@@ -75,7 +75,7 @@
  * non-{@code String} key or value, the call will fail. Similarly,
  * the call to the {@code propertyNames} or {@code list} method
  * will fail if it is called on a "compromised" {@code Properties}
- * object that contains a non-{@code String} key.
+ * object that contains a non-{@code String} key or value.
```

Comment:
Since `propertyNames` won't throw `CCE` with a non-String value, instead of adding the "or value" wording here, we should move the `list` method to the sentence above and remove it from the current sentence.

E.g., on line 73 

> "If the {@code store}, {@code save}, or {@code list) method ...

Reply:
Thanks for the changes. The current form seems reasonable to me, although others might have comments. I created the following CSR, [JDK-8362447](https://bugs.openjdk.org/browse/JDK-8362447) for you. We will need a reviewer to take a look at these changes as well. (Nit: we could use the oxford comma in the class description changes.)